app/dashboard/static/js/app/view-jobs-all.2017.3.5.js

```diff
@@ -433,8 +433,8 @@ require([
                 .columns(columns)
                 .order([4, 'desc'])
                 .languageLengthMenu('jobs per page')
-                .rowURL('/job/%(job)s/branch/%(git_branch)s/')
-                .rowURLElements(['job', 'git_branch'])
+                .rowURL('/test/job/%(job)s/branch/%(git_branch)s/kernel/%(kernel)s/')
+                .rowURLElements(['job', 'git_branch', 'kernel'])
```

Comment:
I don't think this one should be changed, the links to go to a view with all the kernel revisions for a given branch should be fine.